hydra/core/utils.py

```diff
@@ -82,17 +87,35 @@ def filter_overrides(overrides: Sequence[str]) -> Sequence[str]:
     return [x for x in overrides if not x.startswith("hydra.")]
 
 
+def _get_callbacks_for_run_job(hydra_context: Optional[HydraContext]) -> "Callbacks":
+    if hydra_context is None:
+        warnings.warn(
+            message=dedent(
+                """
+            run_job's signature has changed in Hydra 1.1.
+            Support for the old style will be removed in Hydra 1.2.
+            For more info, check https://github.com/facebookresearch/hydra/pull/1581.
```

Comment:
```suggestion
            run_job's signature has changed in Hydra 1.1. Please pass hydra_context.
            Support for the old style will be removed in Hydra 1.2.
            For more info, check https://github.com/facebookresearch/hydra/pull/1581.
```

Reply:
The info link https://github.com/facebookresearch/hydra/pull/1581 should contain some TLDR for what people should do to migrate. Sending them to read a design doc does not feel right.
This can also include links to the updated example plugins.

Reply:
Thanks! I've updated the PR's description to provide more info 
https://github.com/facebookresearch/hydra/pull/1581

Reply:
pro tip:
If you are linking to a specific line in master, that link will become wrong when the file changes later.
You can fix it by linking to a specific revision: while you are on the link to master, hit `y` and the browser link will change.

Reply:
nice!! I've fixed the links in #1581